src/aiida_icon/calculations.py

```diff
@@ -170,6 +171,14 @@ def prepare_for_submission(self, folder: folders.Folder) -> datastructures.CalcI
                     "run_icon.sh",
                 )
             )
+        if "env_file" in self.inputs:
+            calcinfo.local_copy_list.append(
+                (
+                    self.inputs.env_file.uuid,
+                    self.inputs.env_file.filename,
+                    "env_file.sh",
```

Comment:
Consistent with above renaming suggestion:

```suggestion
        if "setup_script" in self.inputs:
            calcinfo.local_copy_list.append(
                (
                    self.inputs.env_file.uuid,
                    self.inputs.env_file.filename,
                    "setup_script.sh",
```